docs/example-dataset.md

```diff
@@ -66,7 +65,9 @@ To register our dataset, you need to specify *(a)* a logical name, *(b)* the pat
 and *(c)* which columns hold the group ID (user ID) and the timestamp.
 
 Here's the command to run, and the output you should get (omitting some details):
-
+<!-- This also needs to have a FROCKET_REDIS_PORT=6380 prefix,  it also fails to complete locally 
```

Comment:
I assume that the docker workers can't "find" the local dataset ?

Reply:
We can (at least optionally) support binding to the local ./data dir

Reply:
please revert this part (as I'll do the port changes) and let's merge the other fixes, thx!

Reply:
Removed, once #50  is merged I'll rebase and add any additional tweaks